src/Agent.Worker/Build/TeeTFCommandManager.cs

```diff
@@ -246,17 +245,13 @@ private string FormatArguments(FormatFlags formatFlags, params string[] args)
                 };
                 string arguments = FormatArguments(FormatFlags.None, args);
                 ExecutionContext.Command($@"{_tf} {arguments}");
-                int exitCode = await processInvoker.ExecuteAsync(
+                await processInvoker.ExecuteAsync(
                     workingDirectory: IOUtil.GetWorkPath(HostContext),
                     fileName: _tf,
                     arguments: arguments,
                     environment: null,
+                    requireExitCodeZero: true,
```

Comment:
I am not sure, the only argument i can think of is if the caller want to provide more context to the error message?